adl

```diff
@@ -51,11 +52,26 @@ reset_vars() { #{{{
 declare -a exceptions=("Bakuman" "Araburu Kisetsu no Otome-domo yo" "Doukyonin wa Hiza" "Tokidoki" "Atama no Ue" "Kobato" "Beelzebub-jou no Okinimesu mama" "Dragon, Ie wo Kau" "Hige wo Soru. Soshite Joshikousei wo Hirou")
 
 check_player() { #{{{
-  # Exit if "$player" is not installed
-  if ! [ -x "$(command -v $player)" ]; then
-    echo "Error: '$player' is not installed." >&2
-    print_options
-    exit 1
+  # Check platform and Exit if "$player" is not installed
+  if ! [ -x "$(command -v $windows_specific_cmd)" ]; then
+    echo "Platform:Linux"
+    if ! [ -x "$(command -v $player)" ]; then
+      echo "Error: '$player' is not installed." >&2
+      print_options
+      exit 1
+    fi
+  else 
+    echo "Platform:Windows"
+    [ "$player" == "mpv" ] && player="mpv.com"
```

Comment:
@Justaus3r this line should take care of it.

Reply:
Yeah you can do that ,we can let users decide which provider they wanna use

Reply:
i am afraid this leads to failure of many providers  😔